### PR TITLE
Task 3 and FinalTask done

### PR DESCRIPTION
**Pull request by pchizhov**
В ходе задания была натренирована нейронная сеть CNN, реализованная с нуля и с помощью фреймворка PyTorch. 

Точность на тестовых данных в сети, написанной на Pytorch составила 0.9. Подобранные параметры: learning rate = 0.1, weight decay = 0.0001, число эпох = 25.

Reviewer: @toshiks

### CODE REVIEW

**Comment by pchizhov** [comment]
В ходе последнего задания была натренирована нейронная сеть для распознавания хот-догов с претренированными слоями. Лучший результат показала модель, у которой тренировался только последний слой с learning_rate=0.001. F1 score, показанный на валидационном датасете, составил 0.939

Review @toshiks
